app/views/layout.erb

```diff
@@ -15,8 +15,12 @@
 <body>
   <div class="container">
     <strong>Together, we can TODO it!</strong>
-    <a href="/users/new">Sign Up</a>
-    <a href="/sessions/new">Sign In</a>
+    <% if session[:user_id] %>
```

Comment:
Creating a `current_user` helper would be an improvement on looking directly at the session here.